src/app/components/grid/index.html

```diff
@@ -86,7 +86,7 @@
       propertyName="sortField"
       isOptional="true"
     >
-    Specifies the field to use to sort the grid. This property accepts a <stache-code>ListSortFieldSelectorModel</stache-code> value that contains a <stache-code>fieldSelector</stache-code> string to represent the current sort field and a <stache-code>descending</stache-code> boolean to indicate whether to sort in descending order.
+    Displays a caret in the column that sorts the grid. This is particularly useful when you programmatically sort data and want to visually indicate how you sort the grid. This property accepts <a stacheRouterLink="." fragment="listsortfieldselectormodel-properties">a <stache-code>ListSortFieldSelectorModel</stache-code> value</a> to represent the current sort field and the sort order.
```

Comment:
"...that sorts the grid" makes me think its actually doing the sorting. Could this say something like "...that indicates which column has been sorted" ?

Reply:
I tweaked this to address the feedback. Let me know if my change works.